fp.js

```diff
@@ -31,4 +37,8 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-var peopleWhoseNamesHaveTheLetterS = people.filter(person => person.name.match("s"))
+// var peopleWhoseNamesHaveTheLetterS = people.filter(person => person.name.match("s"))
+
+var peopleWhoseNamesHaveTheLetterS = people.forEach(function (person){
+  person.name.match("s");
```

Comment:
not quite, since `.forEach` always returns undefined, if you want to initialize an empty array for people with names contains the letter s, then iterate over each person, if they're name contains an s, add it to that array...i.e.

```js
var peopleWhoseNamesHaveTheLetterS = people.forEach(function (person) {
 if (person.name.match("s")) {
   peopleWhoseNamesHaveTheLetterS.push(person)
}
}
```